cmd/wego/app/add/cmd.go

```diff
@@ -30,10 +30,19 @@ var Cmd = &cobra.Command{
 	Use:   "add [--name <name>] [--url <url>] [--branch <branch>] [--path <path within repository>] [--private-key <keyfile>] <repository directory>",
 	Short: "Add a workload repository to a wego cluster",
 	Long: strings.TrimSpace(dedent.Dedent(`
-        Associates an additional application in a git repository with a wego cluster so that its contents may be managed via GitOps
-    `)),
-	Example:       "wego app add .",
-	RunE:          runCmd,
+		Associates an additional application in a git repository with a wego cluster so that its contents may be managed via GitOps
+	`)),
+	Example: `
+  # Add podinfo application to wego control from local git repository
```

Comment:
I think we should remove "podinfo" from the comment since it isn't mentioned in the actual command and just say, "Add application to wego control..."